### PR TITLE
metro and subdiv geocharts

### PR DESCRIPTION
**Pull request by sandikbarr**
#94 and #98

### CODE REVIEW

File: src/app/geo/geochart.map.component.ts [hunk 1]
```diff
@@ -49,19 +52,30 @@ export class GeochartMapComponent implements OnInit, OnChanges, AfterViewInit {
...
-      } else {
-        height = 320;
-      }
+    if (GeochartMapComponent.googleLoaded && google.visualization) {
       const options = {
         displayMode: 'region',
         colorAxis: {colors: this.colors},
         legend: 'none'
       };
+      const { width, height } = this.getMapWidthOrHeight();
```
Comment (pstjean) [inline src/app/geo/geochart.map.component.ts]: This logic and the following two `if` statements could be combined:
`{width: options['width'], height: options['height']} = this.getMapWidthOrHeight();`

Comment (pstjean) [reply src/app/geo/geochart.map.component.ts]: ... or `Object.assign(options, this.getMapWidthOrHeight())`

Debatable whether either of those is an improvement.

File: src/app/shared/table/nested-totals-table.component.ts [hunk 2]
```diff
@@ -56,21 +56,11 @@ export class NestedTotalsTableComponent {
   @Input() nestedData: TotalsTableRow[];
   @Input() nestedDataLoading: boolean;
   @Input() nestedDataLoaded: boolean;
-  @Input() nestedDataError: any;
+  @Input() nestedDataErrorActions: ACTIONS.AllActions[];
   @Input() routerParams: RouterParams;
   @Output() discloseNestedData = new EventEmitter<string>();
   getGroupName = getGroupName;
 
-  get nestedDataRetryAction() {
```
Comment (pstjean) [inline src/app/shared/table/nested-totals-table.component.ts]: Looks like this moved into `selectNested500ErrorReloadActions` in the error selector? Following the chain of assignment to get there took me a little while.

File: src/app/geo/geochart.map.component.ts (hunk 1 above)
Comment (sandikbarr) [reply src/app/geo/geochart.map.component.ts]: good call 👍

File: src/app/shared/table/nested-totals-table.component.ts (hunk 2 above)
Comment (sandikbarr) [reply src/app/shared/table/nested-totals-table.component.ts]: Yes! Sorry! I factored this out. I was thinking about adding a retry button in the map component as well, but it didn't actually feel right when I did it.

**Pull request merged**